internal/tests/cluster/worker_replay_test.go

```diff
@@ -1,64 +0,0 @@
-package cluster
```

Comment:
This is an important test that we need to keep.

Reply:
I have brought this test back but made it work with our changes